openmetadata-ui/src/main/resources/ui/src/pages/LoginPage/SignInPage.tsx

```diff
@@ -214,7 +214,7 @@ const SignInPage = () => {
                     data-testid="email"
                     label={
                       isAuthProviderLDAP
-                        ? t('label.email')
+                        ? t('label.username')
```

Comment:
Hi @ thiagopirex, I checked with @mohityadav766 who is working on the backend side of things and currently, OM doesn't support login via Username for LDAP so we need to revert this label change
```suggestion
                        ? t('label.email')
```

Reply:
Yes, but I cannot have it for others as well right? It's just the label so we want it to be generic for everyone.

Reply:
Ok.

Reply:
email seems specific. even we use "username".  Some generic perhaps instead of user or email or even Username/Email ?